pkgs/by-name/sr/srm-cuarzo/package.nix

```diff
@@ -0,0 +1,63 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  meson,
+  ninja,
+  pkg-config,
+  libdisplay-info,
+  libdrm,
+  libGL,
+  libinput,
+  mesa,
+  seatd,
+  udev,
+}:
+stdenv.mkDerivation (self: {
+  pname = "srm-cuarzo";
+  version = "0.4.0-1";
+  rev = "v${self.version}";
+  hash = "sha256-PWtDSBYw62sfyN2lpd7NY8SbqDrt5IN1rdLZRtDfals=";
+
+  src = fetchFromGitHub {
+    inherit (self) rev hash;
+    owner = "CuarzoSoftware";
+    repo = "SRM";
+  };
+
+  nativeBuildInputs = [
+    meson
+    ninja
+    pkg-config
+  ];
+
+  buildInputs = [
+    libdisplay-info
+    libdrm
+    libGL
+    libinput
+    mesa
+    seatd
+    udev
+  ];
+
+  outputs = ["dev" "out"];
+
+  postPatch = ''
+    sed \
+      -e "s,/usr/include/drm,${libdrm.dev}/include/drm," \
+      -e "s,/usr/include/libdrm,${libdrm.dev}/include/libdrm," \
+      -i src/meson.build
+  '';
+
+  preConfigure = ''
```

Comment:
would you mind adding a comment why we cd into src here since this is non default behavior?